pandas/tests/io/test_sql.py

```diff
@@ -367,6 +375,270 @@ def test_frame3():
     return DataFrame(data, columns=columns)
 
 
+@pytest.fixture
```

Comment:
Does it makes sense for the the engine and connection fixtures to be `scope="module"` so they are not repeatedly created and destroyed?

Reply:
The fixtures also do cleanups after each test. So it's better to have a new engine/conn for each test. Also, they don't seem to add much overhead.